tests/Doctrine/Tests/TestInit.php

```diff
@@ -18,7 +18,3 @@
 });
 
 require_once __DIR__ . '/../../../vendor/autoload.php';
```

Comment:
This file is most likely no longer needed

Reply:
yup :+1: